test/cql-pytest/test_native_functions.py

```diff
@@ -72,3 +72,49 @@ def test_blobas_wrong_size(cql, table1):
     # value 0x0000007b is not a valid binary representation for type bigint".
     with pytest.raises(InvalidRequest, match='blobasbigint'):
         cql.execute(f"SELECT blobasbigint(intasblob(i)) FROM {table1} WHERE p={p}")
+
+# The mintimeuuid() function works with valid "timestamp"-type values, which
+# are 64-bit *signed* integers representing milliseconds since the epoch.
+# In particular, negative timestamps are allowed and should not be rejected.
+# The function tounixtimestamp can be used to extract the timestamp back from
+# the timeuuid. In this test we use "reasonably"-low timestamps to avoid
+# running into problems with extreme values which we'll test in the next test.
+@pytest.mark.parametrize("timestamp", [-1706638779000, -123, 123, 1706638779000])
```

Comment:
nitpick: could be parameterized by 2 values and 2 multipliers: a positive 1 and negative -1.

Reply:
I prefer not to. I want the tests I write to be readable easily, I don't want the reader to have to do these multiplications to understand what is really being tested. I guess it would make sense to do that if there were a lot of different combinations, but if there are only 4, I think it's clearer to just list them.